lib/cli.js

```diff
@@ -24,50 +24,96 @@ module.exports = function broccoliCLI(args) {
     .option('--port <port>', 'the port to bind to [4200]', 4200)
     .option('--host <host>', 'the host to bind to [localhost]', 'localhost')
     .option('--brocfile-path <path>', 'the path to brocfile')
+    .option('--output-path <path>', 'the path to target output folder')
+    .option('--no-watch', 'turn off the watcher')
```

Comment:
I think having `--watch` as a boolean arg would by default also support `--no-watch`.

Also, I think we should add a `--watcher` option which takes a string to specify the type of watcher to use.  In ember-cli we support `polling`, `node`, and `watchman` which is essentially just passed through to `sane` ([see sane README for details](https://github.com/amasad/sane)).

Reply:
going to implement `--watcher` in separate PR

Reply:
`--watch` defaults to `options.watch === undefined`
while
`--no-watch` defaults to `options.watch === true`

so I think it make sense to keep `--no-watch` on `serve` and `--watch` on `build.

Please take a look on commit https://github.com/broccolijs/broccoli/pull/342/commits/0930a3c5d4dbc6d8ad58599368a6217d156795a4